source/about/integrations.rst

```diff
@@ -38,16 +38,16 @@ Bots
 
 You can deploy interactive bots to help users with processes and tasks with Mattermost by  issuing messages to users they can respond to using buttons and dropdown menus. Bots can be used together with apps and plugins. The Hubot open source project, created by GitHub, Inc., is among the most popular of the bot options.
 
-Prebuilt bots are available on the `Mattermost Marketplace <https://mattermost.com/marketplace/>`__, or you can `configure your own bots <https://developers.mattermost.com/integrate/admin-guide/admin-bot-accounts/>`__.
+Prebuilt bots are available on the `Mattermost Marketplace <https://mattermost.com/marketplace/>`__, or you can `configure your own bots <https://developers.mattermost.com/integrate/reference/bot-accounts/>`__.
 
 Learn about `Mattermost Hubot integration (hubot-matteruser on npm) <https://www.npmjs.com/package/hubot-matteruser>`__ and `other open source community bots available <https://integrations.mattermost.com/>`__ or you can `build your own </integrations/cloud-bot-accounts.html>`_.
 
 Custom slash commands
 ---------------------
 
-A `slash commands </channels/run-slash-commands.html>`__ is similar to an `outgoing webhooks <https://developers.mattermost.com/integrate/admin-guide/admin-webhooks-outgoing/>`__, but instead of listening to a channel, it's used as a command tool in a channel.
+A `slash commands <https://developers.mattermost.com/integrate/slash-commands/>`__ is similar to an `outgoing webhooks <https://developers.mattermost.com/integrate/webhooks/outgoing/>`__, but instead of listening to a channel, it's used as a command tool in a channel.
```

Comment:
```suggestion
A `slash command <https://developers.mattermost.com/integrate/slash-commands/>`__ is similar to an `outgoing webhooks <https://developers.mattermost.com/integrate/webhooks/outgoing/>`__, but instead of listening to a channel, it's used as a command tool in a channel.
```